tests/ast/test_ast_driver.py

```diff
@@ -60,10 +58,11 @@ def render(ast_base64: str) -> str:
     """Uses the unparser to render the AST."""
     assert pytest.unparser_jar, "A valid Unparser JAR path must be supplied either via --unparser-jar=<path> or the environment variable SNOWPARK_UNPARSER_JAR"
     res = subprocess.run([
-        UNPARSER_SCRIPT,
```

Comment:
which java version do we require? Maybe good to have a note somewhere, as this is invoked client-side.

Reply:
I believe Scala requires JDK 3. I don't think there's any danger here, although it's possible one of the dependencies is stricter. Let's see what happens when we try to pass CI/CD.